src/Mayaqua/Str.c

```diff
@@ -2062,11 +2062,11 @@ void EnSafeHttpHeaderValueStr(char *str, char replace)
 	length = StrLen(str);
 	while (index < length)
 	{
-		if (str[index] == '\r' || str[index] == '\n')
+		if ((str[index] == '\r' || str[index] == '\n') &&  length - index > 1)

```

Comment:
Why not adapt the while condition? Here both if conditions were changed, although length - index > 2 implies length - index > 1, as 2 > 1.